scripts/db/README.md

```diff
@@ -0,0 +1,83 @@
+# Transfer data from SQLite to Postgres database
+
+1. \* Decrypt SQLite database if it is encrypted.
+
+   ```sh
+   sqlcipher encrypted_simplex_v1_agent.db
+   ```
+
+   ```sql
+   PRAGMA key = 'password';
+   ATTACH DATABASE 'simplex_v1_agent.db' AS plaintext KEY '';
+   SELECT sqlcipher_export('plaintext');
+   DETACH DATABASE plaintext;
+   ```
+
+   Repeat for `simplex_v1_chat.db`.
+
+2. Prepare Postgres database.
+
+   Build `simplex-chat` executable with `client_postgres` flag and run it to initialize new Postgres chat database.
+
+   This should create `simplex_v1` database with `agent_schema` and `chat_schema` schemas, and `migrations` tables populated. Some tables would have initialization data - it will be truncated via pgloader command in next step.
+
+3. Load data from decrypted SQLite databases to Postgres database via pgloader.
+
+   Install pgloader and add it to PATH.
+
+   ```sh
+   SQLITE_DBPATH='simplex_v1_agent.db' POSTGRES_CONN='postgres://simplex@/simplex_v1' POSTGRES_SCHEMA='agent_schema' pgloader --on-error-stop sqlite.load
+
+   SQLITE_DBPATH='simplex_v1_chat.db' POSTGRES_CONN='postgres://simplex@/simplex_v1' POSTGRES_SCHEMA='chat_schema' pgloader --on-error-stop sqlite.load
+   ```
+
+4. Update sequences for Postgres tables.
```

Comment:
tested like this:
```sql
INSERT INTO users DEFAULT VALUES;
```
without this step it failed on first insert, with it - insert passes (fresh try from start, not consecutively - otherwise sequence is updated)

so it wasn't extensive test of this query yet, may encounter issues in some tables it misses for some reason